src/ResourceManager/RecoveryServices.SiteRecovery/Commands.RecoveryServices.SiteRecovery/help/Update-AzureRmRecoveryServicesAsrPolicy.md

```diff
@@ -66,8 +74,20 @@ The **Update-AzureRmRecoveryServicesAsrPolicy** cmdlet updates the specified Azu
 PS C:\> $currentJob = Update-AzureRmRecoveryServicesAsrPolicy -Policy $Policy -ReplicationFrequencyInSeconds 900
 ```
 
+### Example 2
+```
+PS C:\> Update-AzureRmRecoveryServicesAsrPolicy -AzureToAzur] -InputObject $Policy -ReplicationFrequencyInSeconds 900
+```
+
 Starts the update replication policy operation using the specified parameters and returns the ASR job used to track the operation.
```

Comment:
@viverm same comment about the description being moved from the previous example to the current one. Also, there's a typo in the example with `-AzureToAzur]`

Reply:
updated